README.md

```diff
@@ -1,17 +1,73 @@
 # Shardy
 
-Shardy is an MLIR-based partitioning system. It is meant to be dialect
-agnostic and provide extensive user control and debuggability features. It
-includes an axis-based sharding representation, a set of compiler APIs,
-functionality for sharding propagation, and plans for an SPMD partitioner.
+Shardy is an MLIR-based tensor partitioning system for all dialects. Built from
+the collaboration of both the [GSPMD](https://arxiv.org/abs/2105.04663) and
+[PartIR](https://arxiv.org/abs/2401.11202) teams, it incorporates the best of
+both systems, and the shared experience of both teams and users.
+
+Shardy is meant to be dialect agnostic and provide extensive user control and
+debuggability features. It includes an axis-based sharding representation, a set
+of compiler APIs, functionality for sharding propagation, and plans for an SPMD
+partitioner.
+
+## Documentation
+
+Official Documentation: <https://openxla.org/shardy>
 
 For more information see the docs directory.
 
 ## Status
 
-Shardy is a work in progress. Currently the core dialect and c bindings are
-fully functional. The python bindings are under development.
+Shardy is a work in progress. Currently the core dialect and C bindings are
+fully functional. The Python bindings are under development.
 
 ## Contacts
 
-*   For questions, contact the maintainers - maintainers at openxla.org
+Currently we are not accepting contributions directly to the GitHub repository.
+
+If you still wish to contribute, please contact the maintainers - maintainers at openxla.org
+
+## Build instructions
```

Comment:
I was not sure if this was supposed to be included or not, but these were the steps I followed (based on `.github/workflows/build_test_bazel.yml`).

Reply:
I happy to either keep or remove, deferring to you on whether this is useful and relevant for README. I see stablehlo has a similar section, so we could align to them :)